fairlearn/metrics/_balanced_root_mean_squared_error.py

```diff
@@ -11,17 +11,20 @@
 
 
 def balanced_root_mean_squared_error(y_true, y_pred, sample_weight=None):
-    """
+    r"""The mean of the root mean squared error (RMSE) for the positive and negative cases.
```

Comment:
what does this r do?

Reply:
It allows you to have single `\` characters in the \LaTeX blocks rather than having to put `\\` throughout. I only found out from `flake8` itself complaining :-)